modules/n1ql/pages/n1ql-language-reference/from.adoc

```diff
@@ -122,49 +122,49 @@ include::partial$grammar/dql.ebnf[tag=keyspace-ref]
 image::n1ql-language-reference/keyspace-ref.png["Syntax diagram", align=left]
 
 [horizontal.compact]
-keyspace-path:: <<keyspace-path>> icon:caret-down[]
+// keyspace-path:: <<keyspace-path>> icon:caret-down[]
+keyspace-path:: keyspace-path icon:caret-down[]
 keyspace-partial:: <<keyspace-partial>> icon:caret-down[]
 
 Keyspace reference of the data source.
 The identifiers that make up the keyspace reference are not available as {variables-in-scope}[variables in scope of a subquery].
 
-NOTE: If there is a hyphen (-) inside any part of the keyspace reference, you must wrap that part of the keyspace reference in backticks ({backtick}{nbsp}{backtick}).
-Refer to the examples below.
+include::partial$n1ql-language-reference/hyphen-in-keyspace.adoc[tag=keyspace]
 
-[[keyspace-path]]
-==== Keyspace Path
+//[[keyspace-path]]
+//==== Keyspace Path
 
-[source,ebnf]
-----
-include::partial$grammar/dql.ebnf[tag=keyspace-path]
-----
+//[source,ebnf]
+//----
+//include::partial$grammar/dql.ebnf[tag=keyspace-path]
+//----
 
-image::n1ql-language-reference/keyspace-path.png["Syntax diagram", align=left]
+//image::n1ql-language-reference/keyspace-path.png["Syntax diagram", align=left]
```

Comment:
@simon-dew Keyspace path is fundamentally different in elixir. Instead of the "root" keyspace being a bucket, it's now a collection. I.e.- if you only put one word in the path, it'll assume it's a collection name. This cascades up, so 2 words would be assumed to be scope.collection and so on. I've updated the dql ebnfs and descriptions with this new info

Reply:
If the scope is omitted, the _default scope is still used, but it won't assume the _default collection if you pass it an empty FROM clause (i.e no terms).